src/IR.h

```diff
@@ -594,6 +594,12 @@ struct Call : public ExprNode<Call> {
         return make(param.type(), param.name(), args, Image, FunctionPtr(), 0, Buffer<>(), param);
     }
 
+    /** Convenience constructor for indeterminate_expression() */
+    static Expr make_indeterminate_expression(Type type);
```

Comment:
These feel like they belong better as free functions in IROperator.cpp

Reply:
Fair, will fix